app/Http/Controllers/PageController.php

```diff
@@ -454,6 +454,40 @@ public function restoreRevision($bookSlug, $pageSlug, $revisionId)
         return redirect($page->getUrl());
     }
 
+
+    /**
+     * Deletes a revision using the id of the specified revision.
+     * @param string $bookSlug
+     * @param string $pageSlug
+     * @param int $revisionId
+     * @throws NotFoundException
+     * @throws BadRequestException
+     * @return \Illuminate\Http\RedirectResponse|\Illuminate\Routing\Redirector
+     */
+    public function destroyRevision($bookSlug, $pageSlug, $revId)
```

Comment:
`revId` does not match `@param`  comment on line 462

Reply:
Fixed.